react-backend/client/src/components/admin/activitiesLib/admin_helpers.js

```diff
@@ -82,4 +78,4 @@ const adminFetch = method => async (that, url, body) => {
 export const adminPost = adminFetch('POST');
 export const adminGet = adminFetch('GET');
 
-export const checkAdmin = that => adminFetch('POST')(that, '/admin/check');
+export const checkAdmin = that => adminFetch('POST')(that, 'api/admin/check');
```

Comment:
This needs a leading `/` otherwise it will be interpreted as a relative URL. Same for every API route in this PR.